docs/sbpy/spectroscopy/index.rst

```diff
@@ -47,7 +47,7 @@ inverse length.  For convenience, `sbpy` includes a
 
```

Comment:
What's the purpose of lines 33 - 37?  The three lines 35 - 37 are identical as lines 41 - 43.

Reply:
These imports are needed for astropy versions < 5.3 because the next doctest block only executes for versions >=5.3.  I'll add a comment to this effect to the documentation file.